src/Authoring/WinRT.SourceGenerator/DiagnosticHelpers.cs

```diff
@@ -368,97 +367,58 @@ private string SimplifySyntaxTypeString(string syntaxType)
             SpecialType.System_Enum,
         };
 
-        private static readonly Dictionary<string, List<string>> MappedCSharpTypeMethods = new Dictionary<string, List<string>>()
+        private static readonly HashSet<string> MappedCSharpTypes = new HashSet<string>()

```

Comment:
Any chance for both this list and the other, we can use the one already in WinRTTypeWriter.cs.  We can move it to a helper file if needed.  Is there any difference between both lists other than this only has the keys?

Reply:
yes we are sharing now